content/blog/introducing-excalidraw-plus/index.md

```diff
@@ -2,7 +2,7 @@
 title: Introducing Excalidraw+
 date: 2021-05-03
 author: Excalidraw Team
-link: https://twitter.com/excalidraw
+link: https://github.com/orgs/excalidraw/people
```

Comment:
is this the best promo link? Twitter gives us exposure and is more useful to people than a github team link.